PortalSwift/Sources/PortalSwift/Components/PortalWebView.swift

```
@@ -54,6 +55,7 @@ public class PortalWebView: UIViewController, WKNavigationDelegate, WKScriptMess
   ///   - url: The URL the web view should start at.
   ///   - onError: An error handler in case the web view throws errors.
   public init(portal: Portal, url: URL, onError: @escaping (Result<Any>) -> Void) {
+    self.chainId = portal.chainId ?? 11_155_111
```

May make sense to throw instead of defaulting to sepolia